src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -43,6 +43,36 @@ $schedule-button-height: 51px;
     transform: translateY(0);
   }
 
+  &__breadcrumb {
+    width: 100%;
+    height: 45px;
```

Comment:
```suggestion
    height: $breadcrumb-height;
```

Reply:
created Breadcrumbs component, now the height is not needed.
see the changes here: https://github.com/sparkletown/sparkle/pull/2352/commits/90366891c725a5ed617297b35a2917d624ca566a